sdks/python/setup.py

```diff
@@ -87,6 +106,7 @@ def get_version():
     ]),
     setup_requires=['nose>=1.0'],
     install_requires=REQUIRED_PACKAGES,
+    extras_require=EXTRA_PACKAGES,
```

Comment:
Is it necessary to add `extras_require`? If yes, how could we update tests (travis maybe) to test that this is actually working at setup time.